### DISCUSSION TITLE
"Library Update Done" (every 15 seconds...)

### DISCUSSION THREAD

**Discussion by Retrofreak** [rern/rAudio#915] [Bug Reports]
Since latest Update (20220428), while browsing the playlist, every few seconds the focus gets "stolen" by this "Library Update Done" message bottom left. After a reboot it's okay until I force one real library update. It happens on two PIs here and accessing from different browsers. Bug? Have a nice weekend and thanks for checking it sometime ;-)

**Comment by kw-sudo** [comment]
Also here…RPi 3B+ @ rAudio 20220428 (library auto update ‘on’)

**Reply by kw-sudo** [reply]
You can only stop it by disabling ‘Library Auto Update’ in player settings. Enabled, I also have this phenomenon when I call up the empty home page (desktop shortcut, see attachment) and no song(s) inside playlist.

![E49A85B5-8577-4FE5-8CA9-E3C78C225C0B](https://user-images.githubusercontent.com/104136223/166939195-22675b97-7470-4c03-afae-069154dda77f.jpeg)

**Reply by Retrofreak** [reply]
> You can only stop it by disabling ‘Library Auto Update’ in player settings. 

Good tip, thank you!

**Comment by kw-sudo** [comment]
I‘ve just re-enabled it a few minutes ago: the issue is ‘still alive‘. 🫤
